src/index.html

```diff
@@ -11,8 +11,52 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="IMAC picture"
+        class="card__image"
+      />
+
+      <div class="card__description">
+        <p class="card__description_info">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__description_code">Product code: 195434</p>
+      </div>
+
+      <div class="card__review">
+        <div class="stars">
```

Comment:
```suggestion
        <div class="stars stars--4">
```
Consider using modifiers from the previous task

Reply:
but, the stars task was a little bit different, I didn't get you how shall i use such modifier at this task

Reply:
done